documentation/12-integration-testing.md

```diff
@@ -8,6 +8,15 @@ This integration testing feature is built using the [Azure SDK for Python's test
 
 Before using the toolkit's integration testing functionality, you will need to make sure several types of files are in place and properly configured.
 
+### Dependency Modules Installation
+
+Install the dependency modules / packages if you are running the tests outside the docker container. 
+
+Start the dependency installation by navigating to the toolkit root folder in a terminal or command-line interface and running the following command 
+```
+pip install -r requirements.txt
+```
+
 ### Test deployment parameters files
 
 Each of the sample deployments contains a top level *archetype.test.json* test file that is used during integration testing. This file contains placeholders for a number of important fields used during deployments such subscription IDs or user names, and is used as the basis for the actual *azureDeploy.parameters.json* you will have created to use in your actual deployments.
```

Comment:
Actually, can you replace azureDeploy.parameters.json with archetype.json please? azureDeploy.parameters.json is the previous file name.